src/effects/native/balanceeffect.cpp

```diff
@@ -1,7 +1,11 @@
 #include "balanceeffect.h"
 
-namespace {
+#include "util/defs.h"
 
+namespace {
+    double kMaxCorner = 22000; // Hz
```

Comment:
> It is in th anonymous namespace to hide the value from the linker.

Perhaps I asked the wrong question. Why is it in a named variable? It is only used once and its meaning is clear in that context.

> Step by step:

I don't understand what the use case for this would be. If the user wants the dry signal back they can turn down the chain dry/wet knob or return the balance to center.